frontend/packages/console-app/src/components/nodes/node-status.scss

```diff
@@ -0,0 +1,3 @@
+.co-popover--block {
+  display: flex !important;
```

Comment:
Need to do this to override button styles from PF, which sets the display to inline.

Reply:
> Need to do this to override button styles from PF, which sets the display to inline.

It doesn't look like this is PF doing it. It's us:

https://github.com/openshift/console/blob/23b7ac9cf631b2e33663e492c44202d0f099cfe1/frontend/packages/console-shared/src/components/dashboard/dashboard-card/DashboardCardLink.tsx#L6-L17

We should avoid setting `isInline` and then overriding it with CSS.

Reply:
I understand better why you were using `br` now. Wrapping it in a `div` to make it a block element might be OK.

Reply:
oops!

Reply:
@spadgett Tested doesn't work removing `isInline`.
```
.pf-c-button.pf-m-link.pf-m-inline {
    display: inline;
    padding: 0;
}
```
However, the worst part is even if I remove inline:
`.pf-c-button` sets `display: inline-block`. 
So I think the best way to tackle this would be by not making changes to the current state of the PR's CSS. If we remove `isInline` we will have to override two styles manually.